### DISCUSSION TITLE
"Thumbs down" during voting makes people think they are downvoting

### DISCUSSION THREAD

**Discussion by drewhousman1010** [ParabolInc/parabol#5586] [Feedback]
A [user](https://app.hubspot.com/contacts/3888472/contact/20491151) 🔒  says:

> The thumbs down in the voting phase confuses everyone. They think thumbs down is a downvote, not a way to remove votes.

**Comment by avivapinchas** [comment]
This is fixed! It's now a ➕ / ➖
